scripts/provisioning/s3_start

```diff
@@ -49,6 +51,13 @@ def main():
   try:
     if args.service == 's3server':
       # Add action and remove pass
+      if not args.instance:
```

Comment:
Can we add some comments / descriptions of the changes we are doing?
MAybe function headers help